### ISSUE TITLE
Installation fails when server password contains double quote

### ISSUE THREAD

**Issue by dkattan** [LabtechConsulting/LabTech-Powershell-Module#56]
You probably should build up the Arguments using a string array and pass them to Start-Process so it can properly escape them.

```powershell
$Arguments = @()
$Arguments += "/i"
$Arguments += $InstallerFilePath
$Arguments += "/qn"
if($ServerPassword)
{
   $Arguments += $ServerPassword
}

Start-Process -Wait msiexec -ArgumentList $Arguments
```

https://github.com/LabtechConsulting/LabTech-Powershell-Module/blob/69988bc4a931d2450d086c73a049be9997fe4756/LabTech.psm1#L1292

**Comment by DarrenWhite99** [comment]
It is building an argument list, but it is joining them all together immediately. ( -join ' ' ) .
But the password alone is not the argument, it is `SERVERPASS=PASSWORD` .. I don't know if you can get away with `SERVERPASS=` `PASSWORD` as two arguments.

**Comment by dkattan** [comment]
Upon further inspection, the problem is more about escaping double quotes in MSI parameters, and not escaping them for the command line. 

```powershell
$ServerPassword = 'Hello"world'
"SERVERPASS=""$ServerPassword"""
```

```
SERVERPASS="Hello"world"
```

This makes msiexec unable to parse the command line, and if we were to be in an interactive session, pops up the help dialog.

I think the solution instead is a $ServerPassword = $ServerPassword.Replace('"','""') on the preceding line

As an aside, TIL I put entirely too much faith in Start-Process handling of -ArgumentList when it is an array.

https://github.com/PowerShell/PowerShell/blob/9f1f6762073e450d0940cfcbe8ee6dbb8710ea08/src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs#L1876

I naively assumed it was scanning for spaces in each array item and wrapping it in double quotes if found. 

This is not the case. 

```powershell
Start-Process -Wait msiexec -ArgumentList "/i","C:\Mimecast for Outlook 7.9.0.79 (x64).msi","/qn"
```

Fails in the same mode as above, however

```powershell
Start-Process -Wait msiexec -ArgumentList "/i","`"C:\Mimecast for Outlook 7.9.0.79 (x64).msi`"","/qn"
```

Works as expected.

**Comment by DarrenWhite99** [comment]
Resolved by pull #58
